katas/content/marking_oracles/bit_sum_div_3/solution.md

```diff
@@ -1,4 +1,21 @@
-[placeholder for solution text]
+Let's start by considering a simpler version of the problem. How would you check whether the number of bits equal
+to $1$ is divisible by $2$? 
```

Comment:
nit: it seems like there is an unintended extra line here.

Reply:
Renders fine for me, I'm artistically asking a question and letting the reader contemplate before answering

![image](https://github.com/microsoft/qsharp/assets/10113024/7b0d56e6-3102-4705-a652-69797ea6d8a4)